src/Directory.Build.props

```diff
@@ -1,3 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
-  <Import Project="..\build\build.props" />
-</Project>
\ No newline at end of file
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)..\Directory.Build.props" />
```

Comment:
>$(MSBuildThisFileDirectory) [](start = 19, length = 27)

This qualification is not necessary in Imports. The import is already relative to the containing file.

Reply:
fixed